daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestBlob.scala

```diff
@@ -110,19 +111,23 @@ class TestBlob {
    *
    ***/
   @Test def test_2GB_blob(): Unit = {
-    val schema = path("daffodil-cli/src/test/resources/org/apache/daffodil/cli/large_blob.dfdl.xsd")
+    val schema = path(
+      "daffodil-cli/src/test/resources/org/apache/daffodil/cli/large_blob.dfdl.xsd",
+    )
     val input = path("daffodil-cli/src/test/resources/org/apache/daffodil/cli/input/2049MB.bin")
 
     assumeTrue("large test input file must be manually generated", exists(input))
 
     withTempFile { infoset =>
       withTempFile { unparse =>
         withBlobDir {
-          runCLI(args"parse -s $schema -o $infoset $input", timeout = 120) { cli =>
-          } (ExitCode.Success)
+          runCLI(args"parse -s $schema -o $infoset $input", timeout = 120) { cli => }(
+            ExitCode.Success,
+          )
 
-          runCLI(args"unparse -s $schema -o $unparse $infoset", timeout = 120) { cli =>
-          } (ExitCode.Success)
+          runCLI(args"unparse -s $schema -o $unparse $infoset", timeout = 120) { cli => }(
+            ExitCode.Success,
+          )
```

Comment:
This particular changes makes this much less readable. For consistency, we really want all these tests to have the same pattern of

```scala
runCLI(args"...") { cli =>
  // zero or more lines of actual checks
}(ExitCode.Foo)
```

It seems the ones with bad formatting are those without actual CLI expect checks, likely because all the test cares about is the exit code. I don't think there are too many of those--maybe we just add a comment that makes it clear no expect logic is intentional, and it might also prevent scalafmt from changing these lines? For example:

```scala
runCLI(args"...") { cli =>
  // this block intentionally left blank
}(ExitCode.Success)
```

Reply:
Yes, a well-placed comment like "// checks only exit code" in those places probably will preserve the desired formatting.